client/my-sites/site-settings/form-seo.jsx

```diff
@@ -224,7 +227,11 @@ export const SeoForm = React.createClass( {
 		const isDisabled = isSitePrivate || isSubmittingForm || isFetchingSettings;
 		const isSaveDisabled = isDisabled || isSubmittingForm || ( ! showPasteError && invalidCodes.length > 0 );
 
-		const sitemapUrl = `https://${ slug }/sitemap.xml`;
+		const seoTitle = siteDescription.length
+			? `${ siteTitle } | ${ siteDescription }`
```

Comment:
Not sure it matters _too much_, but the separator is customizable by the theme, and defaults to `-` when using the `title-tag` feature:

https://github.com/WordPress/WordPress/blob/b0f64a9/wp-includes/general-template.php#L1017-L1024